packages/api/database/migrations/1590493573504_technology_schema.js

```diff
@@ -0,0 +1,42 @@
+/** @type {import('@adonisjs/lucid/src/Schema')} */
+const Schema = use('Schema');
+
+class TechnologySchema extends Schema {
+	up() {
+		this.create('technologies', (table) => {
+			table.increments();
+			table.string('title').notNullable();
+			table
+				.string('slug')
```

Comment:
Ah...um detalhe que percebi aqui. Eu tava fazendo uns testes manuais. Quando fazia o primeiro cadastro, tudo ok. NO segundo, com os mesmos dados, gerava um erro. Botando no modo de debug, vi que o problema era o `slug` que deveria ser `unique`. Daí percebi algo aqui: o `slug` é unique, mas ele é gerado a partir do `title`, que não é unique. Logo, se 2 tecnologias tiverem o mesmo título (difícil, mas não é impossível), vai gerar um erro na hora de cadastrar esse slug. Não deveria ter algo pra diferenciar do primeiro pro segundo? TIpo, um número no início ou final...?

Reply:
Sim, caso os títulos sejam iguais precisamos adicionar um `-1` após o slug.

Criei uma issue pra isso: https://github.com/ufersa/plataforma-sabia/issues/131